Application/src/model/Parser.java

```diff
@@ -35,60 +41,85 @@
 
 public class Parser {
 
-    /**
+    /*
      * defining two different counter variables that will increment each time the diagram calling method
      * that they are is called in order to differentiate between the imported diagrams
      * These two counters are static in order that they change each time and new diagram is imported from the main method
```

Comment:
since you don't use the old counters anymore, don't forget to delete this comment